pkg/kubectl/cmd/get/get.go

```diff
@@ -219,7 +220,12 @@ func (o *GetOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []stri
 
 	o.IncludeUninitialized = cmdutil.ShouldIncludeUninitialized(cmd, false)
 
-	if resource.MultipleTypesRequested(args) {
+	restConfig, err := f.ToRESTConfig()
+	if err != nil {
+		return err
+	}
+
+	if resource.MultipleTypesRequested(args) || resource.CustomResourceDefinitionRequested(crdclientset.NewForConfigOrDie(restConfig), args) {
```

Comment:
instead of specifically checking if it is a category related to a CRD, I think we should instead check if it is a category after all.

Reply:
something like this? https://github.com/nikhita/kubernetes/commit/1f7ec99f6cbb6aee61b5453c699648dd8f20d438